Inventory.java

```diff
@@ -0,0 +1,148 @@
+package inventory;
+import java.util.Scanner;
+
+import Exceptions.DuplicateProductNameException;
+import Exceptions.InvalidProductNameException;
+import Exceptions.InvalidProductPriceException;
+import Exceptions.InvalidProductQtyException;
+import Exceptions.ProductNotFoundException;
+
+public class Inventory {
+
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		ProductManager productManager = new ProductManager();
+		while(true) {
+			System.out.println("Inventory Management System");
+			System.out.println("1) Add Product");
+			System.out.println("2) Delete Product");
+			System.out.println("3) Update Product");
+			System.out.println("4) Display All Products");
+			System.out.println("5) Exit");
+			int userChoice = sc.nextInt();
+			
+			switch(userChoice) {
+			case 1 : 
+				AddProduct();
+				break;
+			case 2:
+				DeleteProduct();
+				break;
+			case 3:
+				UpdateProduct();
+				break;
+			case 4:
+				productManager.DisplayProducts();
+				break;
+			case 5:
+				System.exit(0);
+				break;
+			default:
+				System.out.println("Invalid choice");
+				break;
+			}
+		}
+	}
+	
+	private static void AddProduct() {
```

Comment:
Don't we use the camelCase naming convention for functions? Take care at other places also.